tests/mocha/comment_deserialization_test.js

```diff
@@ -10,6 +10,7 @@ import {
   sharedTestSetup,
   sharedTestTeardown,
 } from './test_helpers/setup_teardown.js';
+import {CommentIcon} from '../../core/icons/comment_icon.js';
```

Comment:
I believe this should have been `from '../../build/src/core/icons/comment_icon.js'` (here and below).

We are now seeing build warnings that I believe are due to this; I'm actually not certain how this test is running successfully as I would not expect the import as written to work at all:

```
Could not find "../../core/icons/comment_icon.js".Assuming it (and its transitive dependencies) are non-Closure managed. /Users/cpcallen/src/blockly/tests/mocha/comment_deserialization_test.js
Could not find "../../core/icons/comment_icon.js".Assuming it (and its transitive dependencies) are non-Closure managed. /Users/cpcallen/src/blockly/tests/mocha/contextmenu_items_test.js
...
```